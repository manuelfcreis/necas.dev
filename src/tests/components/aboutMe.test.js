import React from "react";
import AboutMe from "../../components/aboutMe";
import { shallow } from "enzyme";

it("renders correctly", () => {
  const element = shallow(<AboutMe />);
  expect(element).toMatchSnapshot();
});
