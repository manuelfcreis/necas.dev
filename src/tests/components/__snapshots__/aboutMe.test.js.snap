// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <Card>
    <h2>
      Manuel Costa Reis
    </h2>
    <ul
      className="iconList"
    >
      <SocialLink
        alt="Twitter"
        image="assets/twitter.png"
        link="https://twitter.com/ManuelCReis"
      />
    </ul>
    <hr />
    <ul
      className="summary"
    >
      <li>
        I'm a
         
        full-stack software developer, data scientist, 
         
        economist and I've had my own start-up, dedicated to podcasts.
      </li>
      <li>
        <strong>
          Data science
        </strong>
         is for me a mixture of both development and economics. I end up gravitating towards it as I enjoy both implementing products in code but also trying to use data as a source for making decisions.
      </li>
      <li>
        I listen to a lot of podcasts and audiobooks.
      </li>
      <li>
        I'm very interested in music. I'm always looking for new things. I'm also very into cinema, host a weekly cinema club and I keep trying to learn more about cinematography and cinema history.
      </li>
      <li>
        Besides all this reading ends up being one of my favorite activities. Both fiction and non-fiction. I try to vary between the two.
      </li>
      <li>
        I have a very large dog. üê∂
      </li>
    </ul>
    <br />
  </Card>
</div>
`;
